components/console/helpers/questionhelper.rst

```diff
@@ -276,11 +276,9 @@ You can also use a validator with a hidden question::
 Testing a Command that Expects Input
 ------------------------------------
 
-If you want to write a unit test for a command which expects some kind of input
-from the command line, you need to set the helper input stream::
+If you want to write a unit test for a command which expects user inputs
+from the command line, you need to set the inputs like this:
```

Comment:
You need to end the paragraph with two colons so that the following block will be properly treated as a PHP code block.